src/modules/fancyzones/lib/FancyZonesData.cpp

```diff
@@ -95,11 +95,24 @@ std::optional<FancyZonesDataTypes::CustomZoneSetData> FancyZonesData::FindCustom
 
 void FancyZonesData::AddDevice(const std::wstring& deviceId)
 {
+    using namespace FancyZonesDataTypes;
+
     std::scoped_lock lock{ dataLock };
     if (!deviceInfoMap.contains(deviceId))
     {
         // Creates default entry in map when ZoneWindow is created
-        deviceInfoMap[deviceId] = FancyZonesDataTypes::DeviceInfoData{ FancyZonesDataTypes::ZoneSetData{ NonLocalizable::NullStr, FancyZonesDataTypes::ZoneSetLayoutType::Blank } };
+        GUID guid;
+        auto result{ CoCreateGuid(&guid) };
+        wil::unique_cotaskmem_string guidString;
+        if (result == S_OK && SUCCEEDED(StringFromCLSID(guid, &guidString)))
+        {
+            DeviceInfoData defaultDeviceInfoData{ ZoneSetData{ guidString.get(), ZoneSetLayoutType::PriorityGrid }, true, 16, 3 };
```

Comment:
Can you replace `true`, `16` and `3` with consts or explicit argument names?

Reply:
Done